<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="CustomDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="VideoPlayerWidget" name="videoPlayerWidget">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <widget class="VideoControl" name="videoControl" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>238</y>
         <width>230</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="VideoProgressBar" name="videoProgress">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>220</y>
         <width>229</width>
         <height>18</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>18</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenUrl"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionOpenUrl">
   <property name="text">
    <string>Open URL</string>
   </property>
  </action>
 </widget>
<customwidgets>
 <customwidget>
  <class>VideoControl</class>
  <extends>QWidget</extends>
  <header>videocontrol.h</header>
  <container>1</container>
 </customwidget>
 <customwidget>
  <class>CustomDockWidget</class>
  <extends>QWidget</extends>
  <header>customdockwidget.h</header>
  <container>1</container>
 </customwidget>
 <customwidget>
  <class>VideoPlayerWidget</class>
  <extends>QFrame</extends>
  <header>videoplayerwidget.h</header>
  <container>1</container>
 </customwidget>
 <customwidget>
  <class>VideoProgressBar</class>
  <extends>QProgressBar</extends>
  <header>videoprogressbar.h</header>
 </customwidget>
</customwidgets>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
